src/docs/user/references/GenericInformation.adoc

```diff
@@ -461,4 +465,13 @@ The <<../JobPlanner/JobPlannerUserGuide.html#_all_doc_jp_user_guide, Job Planner
 
 * `calendar.name`: the calendar name based on which the job is planned.
 * `next.execution`: the next execution date of the planned job. The date is formatted with respect to the pattern `yyyy-MM-dd HH:mm:ss z` and considers the time zone specified in the Job Planner configuration.
-When no time zone is specified, the default time zone of ProActive server is considered.
\ No newline at end of file
+When no time zone is specified, the default time zone of ProActive server is considered.
+
+==== submission.mode
+
+The `submission.mode` Generic Information displays the name of the portal from where the job was submitted.
```

Comment:
.. the name of the portal, service or feature from which the job was submitted.

It can either be one of the standardized values below or a user-defined name.